logic.js

```diff
@@ -0,0 +1,117 @@
+// const postIdCounter = function(posts){
```

Comment:
when you finish your code, make sure you delete all the `//comments` and the `console.log()s`